triggers/service/src/main/resources/com/daml/lf/engine/trigger/db/migration/oracle/V5__Add_read_as.sql

```diff
@@ -0,0 +1,5 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Add application_id to running trigger table defaulting to trigger_instance
+alter table ${table.prefix}running_triggers add (read_as nvarchar2(2000));
```

Comment:
one of these things is not like the other

Reply:
I thought I left a comment here but apparently I haven't: what happens if there is a running trigger and the trigger service undergoes an upgrade on the following restart? Will this additional column be nullable, thus meaning that new rows will have a `null` value for `read_as`? (Which I assume works fine, as `null` seems to be used for empty `read_as` from what I can tell reading at the code)

Reply:
`null = ''` on Oracle; `not null` rejects empty strings, which is not desirable here. In other words the default is correct for Oracle.